content/lib/pencil-tracer.js

```diff
@@ -731,8 +775,37 @@
       }
     };
 
+    JavaScriptInstrumenter.prototype.isFunctionDef = function(node) {
+      var ref;
+      return (node != null ? node.type : void 0) === "FunctionDeclaration" || ((node != null ? node.type : void 0) === "VariableDeclaration" && node.declarations.length === 1 && ((ref = node.declarations[0].init) != null ? ref.type : void 0) === "FunctionExpression");
```

Comment:
Line too long. I don't mind scrolling a little, but if I have to scroll too far, I lose context.